meeting/meeting_september_15_2021.md

```diff
@@ -0,0 +1,64 @@
+# September 15th (7 PM CST)/16th (9 AM JST) Meeting 2021
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Tibo)**
+* **#Query(Lloyd)**
+* **#Query(Chris)**
+* **#Query(Jesse)**
+* **#Query(Matt)**
+* **#Query(Ben)**
+* **#Query(Antoine)**
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+* rust-dlc
+* atomic finance
+* Binary outcome DLC implementation from Lloyd
+* others?
+
+## Specification Writing
+
+**#Status_Update_Interrupt**
+
+* If anyone is interested it would be nice to document loss-of-fund vectors
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/132
+  * Being robust against random failures?
+    * Nothing in specification about what should be backed up when before sending messages
+    * In LN they also have the channel-reestablish message
+* Draft Updating P2P message serialization to include sub-types, optional fields, tlv streams
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/171
+  * Closer to TLVs than the previous draft
```

Comment:
Actually I'd like to discuss which one of the two drafts we should go for. After implementing something a bit close to the TLV version one, I'm not really sure it's the best approach.